components/root/root.tsx

```diff
@@ -593,9 +592,6 @@ export default class Root extends React.PureComponent<Props, State> {
                                 />
                                 <RootRedirect/>
                             </Switch>
-                            <CompassThemeProvider theme={this.props.theme}>
-                                <OnBoardingTaskList/>
-                            </CompassThemeProvider>
                         </>
```

Comment:
nit : we dont need the fragments here anymore

Reply:
nice catch, thanks!